.gitignore

```
@@ -0,0 +1,278 @@
+# Top level
+__pycache__/*
+
+# Global
+*/*/.vscode/*
```

If you want to ignore all nested vscode gitignores then use
```suggestion
**/.vscode/*
```

---

👍